FizzBuzz - single parameter.rb

```diff
@@ -0,0 +1,50 @@
+def fizz_buzz
+
+  puts "Pick a number:"
+  is_fb = gets.chomp
+  is_fb = is_fb.gsub(/,/, '.')
+
+  def print_fizz_buzz(is_fb)
+    if
+      is_fb%3 == 0 && is_fb%5 == 0
+      puts "FizzBuzz"
+    elsif is_fb%3 == 0
+      puts "Fizz"
+    elsif is_fb%5 == 0
+      puts "Buzz"
+    else
+      puts "No FizzBuzz for you, sport!"
+    end
+  end
+
+  begin
+    is_fb = Float(is_fb)
+  rescue
+    puts "Improper value, try again."
+    is_float = false
```

Comment:
you can just call `fizz_buzz` here and not set this flag - gets rid of the conditional below too :)

Reply:
I've tried this approach and it backfired, so I just sealed this with a flag. But I'll look into it again.

Reply:
if it tested, it backfired and you're not comfortable with it - it's fine to leave it as it is :)

Reply:
I wish I could, you've got me thinking too much. ;)